ipatests/test_integration/test_commands.py

```diff
@@ -1170,3 +1170,35 @@ def test_ipa_adtrust_install_with_locale_issue8066(self):
             assert msg2 not in result.stderr_text
         finally:
             bashrc_backup.restore()
+
+    def test_login_wrong_password(self):
+        """Test ipa user login with wrong password
+
+        When ipa user login to machine using wrong password, it
+        should log proper message in /var/log/secure
+
+        related: https://github.com/SSSD/sssd/issues/5139
+        """
+        tasks.kinit_admin(self.master)
+        testuser = 'testuser'
+        tasks.create_active_user(self.master, testuser, 'Secret123')
+
+        # try to login with wrong password
+        sshconn = paramiko.SSHClient()
+        sshconn.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+        try:
+            sshconn.connect(self.master.hostname,
+                            username=testuser,
+                            password='WrongPassword')
+        except paramiko.AuthenticationException:
+            pass
+
+        sshconn.close()
+
+        # check if proper message logged
+        exp_msg = ("pam_sss(sshd:auth): received for user {}: 7"
+                   "(Authentication failure)".format(testuser))
+        result = self.master.run_command(['tail', '/var/log/secure'])
```

Comment:
As per looking https://github.com/SSSD/sssd/issues/5139
It should be running on client right ?

Reply:
@amore17 
the test is doing ssh from controller to master, meaning the sshd logs must be checked on the master.
@mrizwan93 
on Fedora sshd is logging to the journal, not to /var/log/secure as on RHEL (because rsyslog is not shipped by default). The test will have to be adapted.